src/MongoDB/Manager.c

```diff
@@ -191,6 +195,10 @@ static PHP_METHOD(Manager, __construct)
 	}
 
 	phongo_manager_init(intern, uri_string ? uri_string : PHONGO_MANAGER_URI_DEFAULT, options, driverOptions TSRMLS_CC);
+
+	if (intern->client) {
+		php_phongo_set_monitoring_callbacks(intern->client);
+	}
```

Comment:
Just as my comment on the HHVM PR, in earlier discussions we came to the conclusion that we didn't want to remove the apm callbacks at request shutdown, and to be safe, always set them upon Manager creation.